packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -176,5 +193,19 @@ const patternSource = computed(() => {
       }
     }
   }
+
+  &__footer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    gap: var(--onyx-spacing-2xs);
+    color: var(--onyx-color-text-icons-neutral-soft);
+  }
+
+  &__counter {
+    margin-left: auto;
+    display: block;
+    width: max-content;
+  }
```

Comment:
I would consider `justify-content: space-between`, `flex-grow: 1` (both require removing the `v-if="props.message"`) or using `grid` a cleaner solution.

Reply:
But this would mean that there is always an empty `<span>` if there is no message, right?

Reply:
Yes, which would be fine IMHO. I like to avoid margin for placing content, if it can be avoided:

```css
text-align: right;
flex-grow: 1;
```

Reply:
Nice, done :)